cypress/e2e/auth/signup.cy.js

```diff
@@ -0,0 +1,24 @@
+describe('Signup test', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/sign-up');
+  });
+
+  it.only('Login should display error message', () => {
+    cy.get('#username').type('KevinLu');
+    cy.get('#email').type('turnkevin@gmail.com');
+    cy.get('#password').type('admin123');
+    cy.get('.chakra-checkbox__control').click();
```

Comment:
Don't use css selector, use `data-test-id` instead

Reply:
Ok, I will replace all the css selector